core/scripts/lantern/collect/gcp-setup.sh

```diff
@@ -11,15 +11,14 @@ echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" | sud
 wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
 
 # Node apt-key
-curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash -
+curl -sL https://deb.nodesource.com/setup_16.x | sudo -E bash -
 
 # Install dependencies
 sudo apt-get update
 sudo apt-get install -y xvfb nodejs google-chrome-stable google-cloud-sdk git zip
 sudo npm install -g yarn
 
 # Add a lighthouse user
-sudo useradd -m -s $(which bash) -G sudo lighthouse
```

Comment:
Were you running on a fresh machine or re-using one?

Reply:
Pretty sure it was a fresh machine, I went through a few cycles of creating and removing machines named `lantern-collect-instance`

Reply:
That's weird. Doesn't seem like it could work without this line.

The other gcp script does `|| echo "Lighthouse user already exists!"`, let's do that here too.

Reply:
weirdly I got a lot of echoed "Lighthouse user already exists!" messages when running the regular gcp collect script as well. I also couldn't figure out why :)

Reply:
You were both manually destroying the instance in the GCP dashboard each time? IIRC the script will silently-pass the instance-creation command if it already exists with those parameters.

Reply:
Yeah I was manually destroying it